generateSecrets.js

```
@@ -0,0 +1,50 @@
+  if (regex.test(envContent)) {
+    envContent = envContent.replace(regex, `${key}=${value}`);
+  } else {
+    envContent += `\n${key}=${value}`;
+  }
+  return envContent;
+}
+
+envContent = updateEnvContent(envContent, 'JWT_ACCESS_SECRET', accessSecret);
+envContent = updateEnvContent(envContent, 'JWT_REFRESH_SECRET', refreshSecret);
+envContent = updateEnvContent(envContent, 'JWT_RESET_SECRET', resetSecret);
+envContent = updateEnvContent(envContent, 'JWT_CONFIRM_SECRET', confirmationSecret);  // изменено
```

ESLint

[error] Expected linebreaks to be 'CRLF' but found 'LF'.

(linebreak-style)